docs/source/modules.rst

```diff
@@ -5,3 +5,11 @@
    :members:
    :undoc-members:
    :show-inheritance:
+
+:mod:`Read Services`
+----------------------
+
+.. automodule:: odm2api.ODM2.services.readService
```

Comment:
I haven't fixed all the docstring for this module yet. It's gonna be a mess except for one function `getAffiliations`. But I guess for now that's fine. The current docstrings still rendered, at lease now I can see it without going to the actual module python file.

Reply:
You don't need to merge this. I only sent the PR to show you with code that what was wrong was not your docstring :smile:

Reply:
Could you maybe remove this change and keep the other one to `conf.py` so I can merge? Thanks 😄

Reply:
Done!